python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -368,3 +373,29 @@ def get_asset_key(self, dbt_resource_props: Mapping[str, Any]) -> AssetKey:
 @dataclass
 class DbtManifestWrapper:
     manifest: Mapping[str, Any]
+
+
+def validate_translator(dagster_dbt_translator: DagsterDbtTranslator) -> DagsterDbtTranslator:
+    return check.inst_param(
+        dagster_dbt_translator,
+        "dagster_dbt_translator",
+        DagsterDbtTranslator,
+        additional_message=(
+            "Ensure that the argument is an instantiated class that subclasses"
+            " DagsterDbtTranslator."
+        ),
+    )
+
+
+def validate_translator_opt(
```

Comment:
To be in accordance with `opt_inst_param`

```suggestion
def validate_opt_translator(
```